### DISCUSSION TITLE
Who's going? (02/25 Hackathon Team Formation)

### DISCUSSION THREAD

**Discussion by bayareaunicorn** [bluesky-social/atproto#2214] [General Discussion]
Hello looking for people attending the hackathon  on Sunday 02/25 let’s connect as there’s no rules to team formation ahead of time and we only have 3+ hours.

@snarfed @redsolver @ShinoharaTa  @fishcharlie

**Comment by fishcharlie** [comment]
@bayareaunicorn Haven't even heard about this hackathon. Anywhere where I can find more information?

**Reply by bayareaunicorn** [reply]
I'm not sure where I found it at but here's the link to signup..

https://partiful.com/e/AiscT5PsNTIrxNq58M5q?

**Reply by fishcharlie** [reply]
@bayareaunicorn Looks like it's in San Francisco. Sadly I'm not based in San Francisco, so I won't be able to attend.
